app/assets/javascripts/test/snapshots/public/test-bundle/test/libs/nml.spec.js.md

```diff
@@ -1,56 +0,0 @@
-# Snapshot report for `public/test-bundle/test/libs/nml.spec.js`
```

Comment:
Because `yarn remove-snapshots` delete all snapshots. In most cases they are recreated by running the E2E test. The nml spec snapshot is part of a unit test though. Therefore most people forget to run these after deleting the snapshots / while refreshing the E2E test snaspshots.